### PR TITLE
unhardcoded input file size for elastic searchn

### PR DESCRIPTION
**Pull request by AnatoliiStepaniuk**

### CODE REVIEW

File: elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java [hunk 1]
```diff
@@ -12,19 +12,20 @@
...
+            {
+                ElasticSearchAggregator.getInstance().
+                        moveRecordFromElasticSearchToMongo( esRecord, MongoUtils.getCollection() );
+            }
         }
-            // TODO: replace these conditions:
-        if( records.size() == 1 && records.get( 0 ).equals( new MinerRecord(  ) )
-                && ElasticSearchAggregator.getInstance().isElasticSearchFillerFinished() ) // TODO: это фикс для бага - при записи в ES создается еще одна запись со всеми параметрами null
+
+        if( ElasticSearchAggregator.getInstance().isElasticSearchFillerFinished() && records.size() == 1 )
```
Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: what does it mean `records.size() == 1`? Maybe move to another method or add comment

File: elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java [hunk 2]
```diff
@@ -12,19 +12,20 @@
...
         {
             MessageBus.getInstance().sendMessage( ElasticSearchProperties.getInstance().getElasticsearchToMongoIsFinishedKey() );
             System.out.println( "ElasticSearchToMongo :: Successfully finished!");
```
Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: use logging framework

File: elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java [hunk 3]
```diff
@@ -12,19 +12,20 @@
 @Component
 public class ElasticSearchAggregatorScheduledTask
 {
-    // TODO: how can we stop it from running after all records are persisted?
     @Scheduled(fixedDelay = 100)
     public void runWithDelay() throws IOException
     {
         List<MinerRecord> records = ElasticSearchAggregator.getInstance().retrieveAllRecords();
         for( MinerRecord esRecord : records )
```
Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: use stream filtering here

File: elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java [hunk 4]
```diff
@@ -12,19 +12,20 @@
...
         {
-            ElasticSearchAggregator.getInstance().
-                    moveRecordFromElasticSearchToMongo( esRecord, MongoUtils.getCollection() );
+            if( !esRecord.equals( new MinerRecord()) )
```
Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: why comparing to new?

Comment (AnatoliiStepaniuk) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: There's a bug - a record in  ElasticSearch with all fields == null.
Could not delete this record with ElasticSearch API.

File: elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java [hunk 5]
```diff
@@ -19,14 +19,10 @@
...
-        for( MinerRecord esRecord : records )
-        {
-            if( !esRecord.equals( new MinerRecord()) )
-            {
-                ElasticSearchAggregator.getInstance().
-                        moveRecordFromElasticSearchToMongo( esRecord, MongoUtils.getCollection() );
-            }
-        }
+
+        records.stream().filter( record -> !record.equals(new MinerRecord()) )
```
Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: move every stream call to the next line like:

``` java
 records.stream()
    .filter( ... )
    .map( ... )
    .forEach( ... )
```

File: elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java [hunk 6]
```diff
@@ -19,14 +19,10 @@
...
+                .forEach(record -> ElasticSearchAggregator.getInstance().
```
Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: replace singleton with DI

Comment (ardzi007) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: can we do batch update/move here?

Comment (AnatoliiStepaniuk) [inline elastic-search-aggregator/src/main/java/in/ramanujam/elasticsearch/aggregator/ElasticSearchAggregatorScheduledTask.java]: DI integration is a separate branch made by Denys.

**Pull request merged**